packages/core/contracts/Escrow.sol

```diff
@@ -274,12 +276,12 @@ contract Escrow is IEscrow, ReentrancyGuard {
         uint256 reputationOracleFee = 0;
         uint256 recordingOracleFee = 0;
         for (uint256 j; j < _amounts.length; j++) {
-            uint256 singleReputationOracleFee = reputationOracleStake
-                .mul(_amounts[j])
-                .div(100);
-            uint256 singleRecordingOracleFee = recordingOracleStake
-                .mul(_amounts[j])
-                .div(100);
+            uint256 singleReputationOracleFee = uint256(
+                reputationOracleFeePercentage
```

Comment:
again here there is a division and the _amounts array should check that 0 is not acceptable

Reply:
We already have `require(_amounts[i] > 0, 'Amount should be greater than zero');` at line 232.

Reply:
This was updated yes ok